platform/android/app/src/main/kotlin/coop/polypoly/polypod/features/Feature.kt

```diff
@@ -1,20 +1,22 @@
 package coop.polypoly.polypod.features
 
 import FeatureManifest
+import android.content.Context
 import android.graphics.Bitmap
 import android.graphics.BitmapFactory
 import android.graphics.Color
 import android.util.DisplayMetrics
+import coop.polypoly.polypod.PDFBitmap
 import java.util.zip.ZipFile
 
 class Feature(
-    val fileName: String,
+    private val fileName: String,
     val content: ZipFile,
+    val context: Context,
```

Comment:
Is this accessed as such somewhere? Does it need to be public? (Also: content, but that's outside this PR)

Reply:
It is used to get screen measurements, and yes it should be private.